lib/domUpdates.js

```diff
@@ -1,10 +1,15 @@
 const domUpdates = {
   
-  updateScore(score) {
-    // select dom element 
-    // display new score on screen
+  newGame() {
+   $('.catergory-text').text(`Category: ${game.getCategory()}`);
+   $('.round-text').text(`Round: ${game.currentRound}`)
+   $('.pop-up-screen').toggle();
+   $('.player1-text').text(game.players[0].name);
+   $('.player2-text').text(game.players[1].name);
+   $('.player3-text').text(game.players[2].name);
```

Comment:
This looks like a good use-case for a for loop -- you could just do this once and dynamically replace the values in the class name and the index. e.g.:

```
let playerClass = `.player${i + 1}-text`;
$(playerClass).text(game.players[i].name);
```